### PR TITLE
EPMRPP-33145: suite view

### PR DESCRIPTION
**Pull request by DiscoElevator**

### CODE REVIEW

**Comment by ArtsiomYeliseyenka** [comment]
TBD

File: app/src/controllers/launch/actionValidators.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const isAdminOrProjectManager = (state) => {
   const activeProject = activeProjectSelector(state);
   return (
     isAdminSelector(state) ||
-    user.assigned_projects[activeProject].projectRole !== 'PROJECT_MANAGER'
+    user.assigned_projects[activeProject].projectRole === 'PROJECT_MANAGER'
```
Comment (ArtsiomYeliseyenka) [inline app/src/controllers/launch/actionValidators.js]: We have common constants for project roles. Is there possibility exists to use permission_resolver created by Yana?

File: app/src/controllers/launch/actionValidators.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ export const validateMergeLaunch = (launch, launches, state) => {
     return 'selectMoreItems';
   }
   const user = userInfoSelector(state);
-  if (launch.owner !== user.userId && isAdminOrProjectManager(state)) {
+  if (launch.owner !== user.userId && !isAdminOrProjectManager(state)) {
```
Comment (ArtsiomYeliseyenka) [inline app/src/controllers/launch/actionValidators.js]: Is there possibility exists to use permission_resolver created by Yana?

File: app/src/controllers/launch/sagas.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { takeEvery, all, put, select } from 'redux-saga/effects';
+import { fetchDataAction } from 'controllers/fetch';
+import { activeProjectSelector } from 'controllers/user';
+import { FETCH_LAUNCHES, NAMESPACE, FETCH_LAUNCH, CURRENT_LAUNCH_NAMESPACE } from './constants';
+
+function* getLaunches({ payload }) {
+  const activeProject = yield select(activeProjectSelector);
+  const url = `/api/v1/${activeProject}/launch`;
```
Comment (ArtsiomYeliseyenka) [inline app/src/controllers/launch/sagas.js]: Please use URLS.js

File: app/src/controllers/launch/sagas.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  yield put(fetchDataAction(NAMESPACE)(url, payload));
+}
+
+function* watchFetchLaunches() {
+  yield takeEvery(FETCH_LAUNCHES, getLaunches);
+}
+
+function* getLaunch({ payload }) {
+  const activeProject = yield select(activeProjectSelector);
+  const url = `/api/v1/${activeProject}/launch/${payload}`;
```
Comment (ArtsiomYeliseyenka) [inline app/src/controllers/launch/sagas.js]: Please use URLS.js

File: app/src/controllers/suite/sagas.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import { takeEvery, all, put, select } from 'redux-saga/effects';
+import { fetchDataAction } from 'controllers/fetch';
+import { activeProjectSelector } from 'controllers/user';
+import { FETCH_SUITES, NAMESPACE } from './constants';
+
+function* getSuites({ payload }) {
+  const activeProject = yield select(activeProjectSelector);
+  const url = `/api/v1/${activeProject}/item?filter.eq.launch=5af4598e20b5430001bee8a1`;
```
Comment (ArtsiomYeliseyenka) [inline app/src/controllers/suite/sagas.js]: hardcoded launch id? This for development needs?

Comment (DiscoElevator) [reply app/src/controllers/suite/sagas.js]: This will be removed after proper routing implementation.

**Comment by ArtsiomYeliseyenka** [comment]
Please resolve conflicts

**Pull request merged**